mitmproxy/console/__init__.py

```diff
@@ -58,6 +58,8 @@ def update_flow(self, f):
         super(ConsoleState, self).update_flow(f)
         if self.focus is None:
             self.set_focus(0)
+        elif self.follow_focus:
+            self.set_focus(self.flow_count())
```

Comment:
Shouldn't we do the same we do in `add_flow` here? (Maybe we can even factor that out in its own `update_focus` method)